Content.Server/Fluids/EntitySystems/SmokeSystem.cs

```diff
@@ -280,6 +281,8 @@ private void ReactWithEntity(EntityUid entity, EntityUid smokeUid, Solution solu
         var cloneSolution = solution.Clone();
         var availableTransfer = FixedPoint2.Min(cloneSolution.Volume, component.TransferRate);
         var transferAmount = FixedPoint2.Min(availableTransfer, chemSolution.AvailableVolume);
+        if (!blockIngestion && _smokeFilter.AreFilterWorking(entity))
+            transferAmount = transferAmount * 0.3;
```

Comment:
Don't hardcode the filter efficiency, this should be part of the filter component so that each filter/mask can be balanced individually.

Reply:
<img width="324" alt="image" src="https://github.com/space-wizards/space-station-14/assets/152836416/555d97a2-c944-44b0-b6ef-da7044eb93a2">
<img width="521" alt="image" src="https://github.com/space-wizards/space-station-14/assets/152836416/a42c16a4-0183-430d-bbb1-a5cefb57a54f">

<img width="358" alt="image" src="https://github.com/space-wizards/space-station-14/assets/152836416/83c7424e-a942-4c34-a09b-0e0f5604cece">


I don't realy know why that don't work. The mask do nothing.
I try to do the same thing that OnInhaleLocation.

Reply:
>   transferAmount = transferAmount * 0.3;
the 0.3 value should be on the component as a variable not hardcoded in a function.